### PR TITLE
[Admin cabinet] Fix Position and PositionDTOs, Add tests to all Position DTOs

### LINKED ISSUE #5976: [Admin cabinet] Fix Position and PositionDTOs, Add tests to all Position DTOs
Fix nameEN to nameEn
Add tests to all Position DTOs: AddingPositionDt, PositionDto, PositionWithTranslateDto
Change annotation to @Data

Epic: [Admin https://github.com/https://github.com/ita-social-projects/GreenCity/issues/910]

### PR DESCRIPTION
**Pull request by ospodaryk**
[Admin cabinet] Fix Position and PositionDTOs, Add tests to all Position DTOs
https://github.com/ita-social-projects/GreenCity/issues/5976

Fixed nameEN to nameEn
Added tests to all Position DTOs: AddingPositionDt, PositionDto, PositionWithTranslateDto
Changed annotation to @DaTa

### CODE REVIEW

File: service-api/src/main/java/greencity/dto/position/PositionAuthoritiesDto.java [hunk 1]
```diff
@@ -1,22 +1,11 @@
 package greencity.dto.position;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
-import lombok.ToString;
+import lombok.*;
```
Comment (RomaMocherniuk) [inline service-api/src/main/java/greencity/dto/position/PositionAuthoritiesDto.java]: please do not use `*`

File: service-api/src/test/java/greencity/dto/position/AddingPositionDtoTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.lang.reflect.Field;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class AddingPositionDtoTest {
+    void nameRegex(String name, boolean validates) throws NoSuchFieldException {
```
Comment (RomaMocherniuk) [inline service-api/src/test/java/greencity/dto/position/AddingPositionDtoTest.java:11]: can be `private`

File: service/src/main/java/greencity/mapping/employee/PositionDtoWithTranslateMapper.java [hunk 3]
```diff
@@ -25,7 +25,7 @@ public class PositionDtoWithTranslateMapper extends AbstractConverter<Position,
     protected PositionWithTranslateDto convert(Position position) {
         Map<String, String> nameTranslations = new HashMap<>();
         nameTranslations.put("ua", position.getName());
-        nameTranslations.put("en", position.getNameEN());
+        nameTranslations.put("en", position.getNameEn());
```
Comment (RomaMocherniuk) [inline service/src/main/java/greencity/mapping/employee/PositionDtoWithTranslateMapper.java:27]: please remove unused import` import greencity.dto.position.PositionDto;`

File: service-api/src/main/java/greencity/dto/position/PositionDto.java [hunk 4]
```diff
@@ -5,18 +5,13 @@
...
 import javax.validation.constraints.Pattern;
 
-@Getter
-@Setter
-@NoArgsConstructor
-@AllArgsConstructor
 @Builder
-@EqualsAndHashCode
-@ToString
+@Data
```
Comment (RomaMocherniuk) [inline service-api/src/main/java/greencity/dto/position/PositionDto.java:9]: please do not use `*`

File: service-api/src/main/java/greencity/dto/position/AddingPositionDto.java [hunk 5]
```diff
@@ -5,16 +5,12 @@
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Pattern;
 
-@Getter
-@Setter
-@NoArgsConstructor
-@AllArgsConstructor
 @Builder
-@ToString
+@Data
```
Comment (RomaMocherniuk) [inline service-api/src/main/java/greencity/dto/position/AddingPositionDto.java:9]: please do not use `*`

File: service-api/src/test/java/greencity/dto/position/PositionWithTranslateDtoTest.java [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PositionWithTranslateDtoTest {
+
+    void idValidation(Long id, boolean validates) throws NoSuchFieldException {
```
Comment (RomaMocherniuk) [inline service-api/src/test/java/greencity/dto/position/PositionWithTranslateDtoTest.java:14]: can be `private`

**Review by RomaMocherniuk — APPROVED** [review]

**Review by Markiro1 — APPROVED** [review]

**Review by ABbondar — APPROVED** [review]

**Review by BranetE — APPROVED** [review]

**Pull request merged**